package.json

```diff
@@ -41,11 +41,15 @@
     ]
   },
   "scripts": {
+    "modules": "yarn --frozen-lockfile --prefer-offline",
+    "modules-prod": "yarn --frozen-lockfile --prefer-offline --prod",
     "docs": "documentation readme lib/** --config documentation.yml --section=API",
-    "build": "tsc",
     "dev": "tsc --watch",
     "test": "jest --runInBand",
-    "release": "standard-version"
+    "release": "standard-version",
+    "build": "rm -rf lib/ && tsc",
```

Comment:
I hear often that `rm` won't work on windows; a common suggestion is the use [rimraf](https://www.npmjs.com/package/rimraf), which is cross-platform. Not sure how relevant this advice still is given things like WSL but figured I'd mentioned it.

Reply:
I think I'd rather not support windows than import another library right now. we can make a separate ticket out of windows support later. I think there are a bunch of reasons this won't work on windows, not just the building.